element_calcium_imaging/scan.py

```diff
@@ -641,17 +643,19 @@ def estimate_nd2_scan_duration(nd2_scan_obj):
                 )
 
                 self.Field.insert(
-                    dict(
-                        key,
-                        field_idx=0,
-                        px_height=pvscan_info["height_in_pixels"],
-                        px_width=pvscan_info["width_in_pixels"],
-                        um_height=pvscan_info["height_in_um"],
-                        um_width=pvscan_info["width_in_um"],
-                        field_x=pvscan_info["fieldX"],
-                        field_y=pvscan_info["fieldY"],
-                        field_z=pvscan_info["fieldZ"],
-                    )
+                    [
```

Comment:
instead of turning this into a list of a single dictionary, use `.insert1()`

Reply:
I didn't have it in this format originally but noticed the ingestion for the other acquisition softwares (ex. lines 463-482) use this format for the Field part table